src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$blue-elements-color: #00acdc;
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$white-button-color: #fff;
```

Comment:
Consider using a more descriptive variable name, such as '$button-background-color' instead of '$white-button-color'